9jaswag/PostIt
# #152541355 Separate Test Mock Data

#### What does this PR do?
- separates mock data for server and client tests into separate files

#### Description of Task to be completed?
- Separate Test Mock Data

#### How should this be manually tested?
- check the ```server/test/data``` directory to see the files for server test mock data
- check the ```client/__mocks__/mockData.js``` file to see the client test mock data

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
- #152541355

#### Screenshots (if appropriate)
- None

#### Questions:
- None
